Description: Builds against Debian packaged google-mock
 and libgtest-dev
Author: Joshua Honeycutt <joshua.honeycutt@gmail.com>
Forwarded: not-needed
Last-Update: 2016-08-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/CMakeLists.txt
+++ b/src/test/CMakeLists.txt
@@ -15,13 +15,11 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 include_directories(
-	../../ext/gtest-1.6.0
-	../../ext/gtest-1.6.0/include
-	../../ext/gmock-1.6.0
-	../../ext/gmock-1.6.0/include)
-	
-add_library(gtest STATIC ../../ext/gtest-1.6.0/src/gtest-all.cc)
-add_library(gmock STATIC ../../ext/gmock-1.6.0/src/gmock-all.cc)
+	/usr/src/gtest
+	/usr/src/gmock)
+
+add_library(gtest STATIC /usr/src/gtest/src/gtest-all.cc)
+add_library(gmock STATIC /usr/src/gmock/src/gmock-all.cc)
 
 if (UNIX)
 	# ignore warnings in gtest and gmock
--- a/src/test/integtests/CMakeLists.txt
+++ b/src/test/integtests/CMakeLists.txt
@@ -56,8 +56,6 @@
 include_directories(
 	../../
 	../../lib/
-	../../../ext/gtest-1.6.0/include
-	../../../ext/gmock-1.6.0/include
 )
 
 if (UNIX)
--- a/src/test/unittests/CMakeLists.txt
+++ b/src/test/unittests/CMakeLists.txt
@@ -51,9 +51,6 @@
 include_directories(
 	../../
 	../../lib/
-	../../../ext/gtest-1.6.0/include
-	../../../ext/gmock-1.6.0/include
-	../../../ext
 )
 
 if (UNIX)
--- a/ext/toolchain/commands1.py
+++ b/ext/toolchain/commands1.py
@@ -457,8 +457,10 @@
 		# if not visual studio, use parent dir
 		sourceDir = generator.getSourceDir()
 
-		self.checkGTest()
-		self.checkGMock()
+                # Disable tests for included gtest/gmock zip files
+                # since we build with Debian's gtest/gmock libs
+		#self.checkGTest()
+		#self.checkGMock()
 		
 		if extraArgs != '':
 			cmake_args += ' ' + extraArgs
